distributed/client.py

```diff
@@ -660,11 +660,11 @@ def __init__(
                 security = getattr(self.cluster, "security", None)
 
         if security is None:
-            security = Security()
+            security = TLSSecurity()
         elif security is True:
-            security = Security.temporary()
+            security = TLSSecurity.temporary()
             self._startup_kwargs["security"] = security
-        elif not isinstance(security, Security):
+        elif not isinstance(security, TLSSecurity):
```

Comment:
My guess is that these type checks should remain against the `Security` type rather than `TLSSecurity`.  Presumably if the user is providing some other Security type then that should be ok too.

Reply:
this should be resolved with commit be7bd57.